logger/logger.go

```diff
@@ -8,8 +8,8 @@ import (
 	"github.com/amp-labs/cli/flags"
 )
 
-func Info(msg string) {
-	fmt.Println(msg)
+func Info(a ...any) {
+	fmt.Println(a...)
```

Comment:
Why not just use `fmt.Println` directly?

Reply:
We'd like to centralize all output to this logger module, which will give us more flexibility in the future to add new behaviors. (e.g. `--debug` flag will pipe all output to a file in addition to printing them, or more colorful output, etc.)